src/sql/src/parsers/create_parser.rs

```diff
@@ -1367,11 +1368,11 @@ PARTITION BY RANGE COLUMNS(b, a) (
   PARTITION r3 VALUES LESS THAN (MAXVALUE, MAXVALU),
 )
 ENGINE=mito";
-        let result = ParserContext::create_with_dialect(sql, &GenericDialect {});
+        let result = ParserContext::create_with_dialect(sql, &GreptimeDbDialect {});
         assert!(result
             .unwrap_err()
             .to_string()
-            .contains("Please provide an extra partition that is bounded by 'MAXVALUE'."));
+            .contains("Expected a concrete value, found: MAXVALU"));
```

Comment:
`MAXVALU` without the last `E`

Reply:
Yes， the `E` is missing in SQL, but the error message is changed after we change the dialect.